doc/howto.rst

```diff
@@ -72,21 +72,25 @@ monitoring data via Linux `perf <https://perf.wiki.kernel.org/index.php/Main_Pag
 kernel subsystem. There are few levels of statistics data being managed by access
 rights. For this demo purposes we collect i915 global metrics, i.e. those
 which describe the whole GPU rather than particular running process. Such
-metrics require CAP_SYS_ADMIN or ``/proc/sys/kernel/perf_event_paranoid<=0``to be
-collected. So, run container as follows to allow GPU metrics collection::
+metrics require CAP_PERFMON or ``/proc/sys/kernel/perf_event_paranoid<=0`` to
```

Comment:
Can we mention from which vanilla kernel this cap has appeared?

Reply:
According to capabilities man page, since Linux 5.8 (released in August 2020): https://man7.org/linux/man-pages/man7/capabilities.7.html